### PR TITLE
Replaced omdb with tmdb

### PR DESCRIPTION
**Pull request by aandis**
@rohitpaulk have a look. Haven't changes specs. Might want to remove the old ones.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -54,7 +54,7 @@ group :development, :test do
   gem 'sqlite3'
 
   gem 'vcr'
-  gem 'webmock'
+  # gem 'webmock'
```
Comment (rohitpaulk) [inline Gemfile]: Why?

Comment (aandis) [inline Gemfile]: Sorry that was for local testing. Forgot to change it back.

File: app/models/movie.rb [hunk 2]
```diff
@@ -9,24 +9,50 @@ def self.from_title(title)
...
-end
+
+  def self.create_movie_from_api(api_movie)
+      movie = Movie.create!(
+        title:       api_movie.original_title,
+        year:        api_movie.release_date,
+        plot:        api_movie.overview,
+        imdb_id:     api_movie.id,  #This isn't imdb id. TODO
+        imdb_rating: api_movie.vote_average,  #Not imdb rating. TODO
+        poster_url:  Enceladus::Configuration::Image.instance.url_for(
```
Comment (rohitpaulk) [inline app/models/movie.rb]: What does this do?

Comment (aandis) [inline app/models/movie.rb]: Takes the movie object received from the api and creates the movie according the attributes of our model.

**Pull request merged**